test/wampy-common-test.js

```diff
@@ -1841,7 +1841,7 @@ serializers.forEach(function (item) {
             });
 
             it('checks options during canceling RPC invocation', function (done) {
-                let reqId;
+                let reqId;  // eslint-disable-line prefer-const
```

Comment:
It seems the rule has a config option that already disables cases like this one:
https://eslint.org/docs/latest/rules/prefer-const#ignorereadbeforeassign

I would prefer setting this to true over adding this comment, what do you think?

Reply:
Oh, Indeed! Agree! Setting is better then inline comment!